src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Providers/TemplateCollectionProviderTests.cs

```diff
@@ -208,7 +211,7 @@ public async Task GivenDefaultTemplateReference_WhenGetTemplateCollectionFromTem
             ImageInfo imageInfo = ImageInfo.CreateFromImageReference(imageReference);
             var newTemplateCollectionProvider = new TemplateCollectionProvider(imageInfo, _emptyClient, _cache, _defaultConfig);
             var templateCollection = await newTemplateCollectionProvider.GetTemplateCollectionAsync();
-            Assert.True(expectedTemplatesCounts <= templateCollection.First().Count());
+            Assert.Equal(expectedTemplatesCounts, templateCollection.First().Count());
```

Comment:
In previous discussion, it looks that we don't need to assert exact equal, as templates are evolving all the time.

Reply:
Details could be found in this PR: https://github.com/microsoft/FHIR-Converter/pull/360/files

Reply:
Thanks, changed it back to "<=".

Reply:
Thanks, changed it back to "<=".